### PR TITLE
refactor!: separate worker from component

### PR DESCRIPTION
**Pull request by brucexc**
## Summary
1. add rsshub as a worker
2. add related content about the rsshub worker in the `/networks` API
3. add resource estimates for the Farcaster worker

The return structure of the /workers_status API has undergone a destructive change

#### before
```json
{
  "data": [
    {
      "network": "savm",
      "worker": "core",
      "tags": null,
      "platform": "Unknown",
      "status": "Ready"
    }
  ]
}
```

#### after
```json
{
  "data": {
    "decentralized": [
      {
        "network": "ethereum",
        "worker": "core",
        "tags": null,
        "platform": "Unknown",
        "status": "Unhealthy"
      },
    ],
    "rss": [
      {
        "network": "rss",
        "worker": "rsshub",
        "tags": [
          "rss"
        ],
        "platform": "Unknown",
        "status": "Unhealthy"
      }
    ],
    "federated": null
  }
}
```

## Checklist

- [x] The commit message follows [Angular Contributing guidelines](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit);
- [x] Tests for the changes have been added (for bug fixes / features);

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: schema/worker/rss/worker.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/rss3-network/protocol-go/schema/tag"
+)
+
+//go:generate go run --mod=mod github.com/dmarkham/enumer@v1.5.9 --values --type=Worker --linecomment --output worker_string.go --json --yaml --sql
+type Worker int
+
+const (
+	Rsshub Worker = iota + 1 // rsshub
```
Comment (pseudoyu) [inline schema/worker/rss/worker.go:12]: better use official name `RSSHub`?

File: internal/node/monitor/monitor_mock.go [hunk 2]
```diff
@@ -27,6 +27,18 @@ func (m *Monitor) MonitorMockWorkerStatus(ctx context.Context, currentState Chec
 		}(w)
 	}
 
+	for _, w := range m.config.Component.RSS {
+		wg.Add(1)
+
+		go func(w *config.Module) {
+			defer wg.Done()
+
+			if err := m.processRssWorker(ctx, w); err != nil {
```
Comment (pseudoyu) [inline internal/node/monitor/monitor_mock.go:36]: same as previous one, use all caps format `RSS` in func name?

**Comment by pseudoyu** [comment]
lgtm

**Review by pseudoyu — APPROVED** [review]

**Comment by brucexc** [comment]
will affect our prod deployment @incubator4

**Pull request merged**